instaclustr/resource_firewall_rule.go

```diff
@@ -115,10 +132,17 @@ func resourceFirewallRuleDelete(d *schema.ResourceData, meta interface{}) error
 
 		rules = append(rules, RuleType{Type: aRule})
 	}
-
-	rule := FirewallRule{
+	var rule FirewallRule
```

Comment:
[Desired, simplicity] - Could this be simplified to -

```
	var rule = FirewallRule{
		Network: d.Get("rule_cidr").(string),
		SecurityGroupId: d.Get("rule_security_group_id").(string),
		Rules:   rules,
	}
```

Reply:
If it still omits the parameter from the json request if its set to "" then i should be able to simplify.